docs/src/literate/quasi_incompressible_hyperelasticity.jl

```diff
@@ -11,60 +11,65 @@
 # In this example we study quasi- or nearly-incompressible hyperelasticity using the stable Taylor-Hood approximation. In spirit, this example is the nonlinear analogue of
 # [`incompressible_elasticity`](@__NBVIEWER_ROOT_URL__/examples/incompressible_elasticity.ipynb) and the incompressible analogue of
 # [`hyperelasticity`](@__NBVIEWER_ROOT_URL__/examples/hyperelasticity.ipynb). Much of the code therefore follows from the above two examples.
-# The problem is formulated in the undeformed or reference configuration with the displacement `u` and pressure `p` being the unknown fields. We now briefly outline
+# The problem is formulated in the undeformed or reference configuration with the displacement $\mathbf{u}$ and pressure $p$ being the unknown fields. We now briefly outline
 # the formulation. Consider the standard hyperelasticity problem
 #
 # ```math
-#   u = \argmin_{v\in\mathcal{K}}\Pi(v),\quad \text{where}\quad \Pi(v)  = \int_\Omega \Psi(v) \ \mathrm{d}\Omega\ .
+#   \mathbf{u} = \argmin_{\mathbf{v}\in\mathcal{K}(\Omega)}\Pi(\mathbf{v}),\quad \text{where}\quad \Pi(\mathbf{v})  = \int_\Omega \Psi(\mathbf{v}) \ \mathrm{d}\Omega\ .
 # ```
+#
+# where $\mathcal{K}(\Omega)$ is a suitable function space.
+#
 # For clarity of presentation we ignore any non-zero surface tractions and body forces and instead consider only
 # applied displacements (i.e. non-homogeneous dirichlet boundary conditions). Moreover we stick our attention to the standard Neo-Hookean stored energy density
 #
 # ```math
-#     \Psi(u) = \frac{\mu}{2}\left(I_1 - 3 \right) - \mu \log(J) + \frac{\lambda}{2}\left( J - 1\right){}^2,
+#     \Psi(\mathbf{u}) = \frac{\mu}{2}\left(I_1 - 3 \right) - \mu \log(J) + \frac{\lambda}{2}\left( J - 1\right){}^2,
 # ```
-# where $I_1 = F:F = F_{ij}F_{ij}$ and J = det(F) denote the standard invariants of the deformation gradient tensor $F = \mathbb{I}+\nabla u$.
+# where $I_1 = \tf{\mathbf{C}} = F_{ij}F_{ij} = \tr(F)$ and $J = \det(\mathbf{F})$ denote the standard invariants of the deformation gradient tensor $\mathbf{F} = \mathbf{I}+\nabla \mathbf{u}$.
```

Comment:
This doesn't render (https://ferrite-fem.github.io/Ferrite.jl/previews/PR439/examples/quasi_incompressible_hyperelasticity/). I guess `\tf` and/or `\tf` isn't available.

Reply:
Shouldn't it be `\tr` on the first? The last step I don't get, perhaps \tr(F^T F)?

Reply:
Right, it is the trace and especially trace of F^T F, as Knut notes. "tf" is a typo.